cfgov/v1/blocks.py

```diff
@@ -85,3 +96,49 @@ class Meta:
 
     class Media:
         js = ["email-signup.js"]
+
+
+class RichTextBlockWithFootnotes(WagtailFootnotesRichTextBlockWithFootnotes):
+    def render_footnote_tag(self, index):
+        template = get_template(settings.WAGTAIL_FOOTNOTES_REFERENCE_TEMPLATE)
+        return template.render({"index": index})
+
+    def replace_footnote_tags(self, value, html, context=None):
+        # This is a wholesale copy of the replace_footnote_tags() method in
+        # wagtail-footnotes's RichTextBlockWithFootnotes. It modifies the
+        # embedded replace_tag() function to call our own
+        # render_footnote_tag() method. This is a change that should be
+        # contributed back upstream to allow straight-forward modification of
+        # footnote link rendering.
+        #
+        # There is an alternative implementation proposed in a PR in 2022:
+        #   https://github.com/torchbox/wagtail-footnotes/pull/27
+        # But I think I prefer providing a new method to a new embedded func.
```

Comment:
As an FYI, I've opened https://github.com/torchbox/wagtail-footnotes/pull/70 with changes that mirror what I've implemented here, so that hopefully, with feedback, better custom rendering support can be added upstream and  we can remove this custom subclass of `RichTextBlockWithFootnotes`.